dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeMetadataConnector.java

```diff
@@ -149,14 +149,14 @@ protected void addSqlTasksWithInfoSchemaFallback(
   }
 
   @ForOverride
-  protected void addSingleSqlTask(
+  protected void addSingleSqlTaskForAssessment(
```

Comment:
Why rename? It looks like it still works for both Assessment and non-Assessment.

Reply:
This has the assessment flags set to `true`. I didn't want to add yet another parameter to the function.

Reply:
Ah so this method was already only used for assessment tasks and not being overriden by any method. If this is the case then we can remove "ForOverride"